layers/b2qt/recipes-bsp/lsr/lsfsync/lsfsync.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Move the service file to the lsfsync repo?

Reply:
No. IMHO .service files should be provided when packaging which means in the yocto-layer in this case. The reason I believe so is that where the binary that the service file refers to is placed is decided at the packaging stage. Meaning, in the repo one does not know where the binary will be locate and should not enforce the placement upon the packaging party.

Reply:
> IMHO .service files should be provided when packaging 

Definitely not.

Normally the service file is generated from a template that has e.g. @bindir@ and other template variables in it if needed.

But just having it here is perhaps easier in this case.

(But also... doing it this way is a reminder/remnant of the past dark (sysv init) days when distro packagers had to write their own init scripts. ;)

Reply:
Hmm maybe you have a point ...
As painful as it is maybe I should try to learn from this :)
Also either your smiley  is incomplete or you are missing a closing parenthesis :angry:

Reply:
I have decided that this is future @DunderRoffe s problem. So lets fix it outside of this PR. Adding a build system for lsfsync is too much to do before merging this.
Are you OK with doing it later ? @martin-ejdestig

Reply:
"It's not just good, it's good enough." (Have not requested changes. :)